src/components/GoalPage/GoalPage.tsx

```diff
@@ -482,6 +483,18 @@ export const GoalPage = ({ user, ssrTime, params: { id } }: ExternalPageProps<{
                         ))}
                     </IssueMeta>
 
+                    <IssueMeta title={tr('Tags')}>
```

Comment:
1. We don't need to show this option if goal doesn't have tags.
2. It would be better to show plus button (same as participants) to provide possibility to add tags inline.